modules.json

```diff
@@ -366,5 +366,30 @@
             "list": "usercontribs",
             "ucuser": "Jimbo Wales"
     	}
-    }
-]
\ No newline at end of file
+    },
+	{
+		"filename": "blocks.py",
+		"docstring": "Demo of `Blocks` module: GET request to list recent blocks",
+		"endpoint": "https://en.wikipedia.org/w/api.php",
+		"params": {
+			"action": "query",
+			"format": "json",
+			"list": "blocks",
+			"bklimit": "3",
+			"bkprop": "id|user|by|timestamp|expiry|reason|range|flags"
+		}	
+	
+	},
+	{
+		"filename": "setlanguage.py",
```

Comment:
Parameters related to `setpagelanguage` module will not go in the `modules.json` file. Auto-generating code samples process currently support only those modules that are `GET` only.

Reply:
Sorry my bad!